doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -579,6 +579,49 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
    :start-after: fota_upgrades_zigbee_start
    :end-before: fota_upgrades_zigbee_end
 
+.. _ug_nrf5340_image_slot_numbering:
+
+Image IDs and slot numbering
+****************************
+
+Each image in the DFU process on the nRF5340 DK is identified by a unique ID, corresponding to a specific memory slot for each core.
+The slots are numbered from 0, and each slot corresponds to a specific location in memory.
+This configuration typically follows:
+
++-------+-------------------------------------------------------------+-----------+-----------+
+| Slot  | Description                                                 | DTS label | n <image> |
++=======+=============================================================+===========+===========+
+| slot0 | First image's primary slot (internal flash)                 | image-0   | 1         |
++-------+-------------------------------------------------------------+-----------+-----------+
+| slot1 | First image's secondary slot (internal or external flash)   | image-1   | 0, 2      |
++-------+-------------------------------------------------------------+-----------+-----------+
+| slot2 | Second image's primary slot (internal RAM)                  | image-2   | 3         |
++-------+-------------------------------------------------------------+-----------+-----------+
+| slot3 | Second image's secondary slot (internal or external flash)  | image-3   | 4         |
++-------+-------------------------------------------------------------+-----------+-----------+
+
+.. note::
+   Slot 1 is the default upload target for firmware updates using the mcumgr CLI or similar software.
+   This behavior applies when no image number is provided and the application is build with the ``CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD`` Kconfig option enabled.
+
+   For non-default behavior targeting the secondary slot (other than slot 1), use ``-n <n>`` from the provided table.
+   You can not, however, upload to the same slot your application is currently running from.
+   For example, if your application is running from the slot 0 (``-n 1``), uploading to that slot will be refused.
+   This slot-to-n numbering adjustment ensures the compatibility with the existing client software, which typically sends the application image to slot 1 without specifying the slot number.
+
+For the nRF5340 DK, serial recovery differs based on the operation mode:
```

Comment:
I suggest that we make it very clear that there are two alternatives: Direct upload in app and direct upload in serial recovery.
Firstly, I suggest that me mention CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD in the former and CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD in the latter.
Secondly, I suggest that we make sub-headers if possible.

Reply:
Given that I am correct that CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD and/or CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD needs to be set?

Reply:
> Firstly, I suggest that me mention CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD in the former and CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD in the latter.
> Secondly, I suggest that we make sub-headers if possible.

Okay, can do, but just want to confirm, there are both the same? Would it be `CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD` for the former, and `CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD` for the latter?

Reply:
I am still confused about when CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD is needed? Cause from what I read in comments on this PR, it could look like CONFIG_MCUMGR_GRP_IMG_DIRECT_UPLOAD is not needed by default. Then when is it needed, if ever? 
If it is not needed for the nRF5340, we probably do not need to mention it here. If it is needed for the nRF5340, we should state why it is needed.

Reply:
For CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD, it is definitely needed, as seen [here](https://github.com/nrfconnect/sdk-nrf/blob/f3a4bcb7148e4e6ade7ec5014f93ba87aba5ac06/modules/mcuboot/boot/zephyr/Kconfig#L144). So I think we should mention this for Serial Recovery, and make it clear that it is for Serial Recovery.